tox.ini

```diff
@@ -68,7 +68,8 @@ allowlist_externals = bashate
 commands = bashate --verbose {toxinidir}/build.sh {toxinidir}/tools/test/functest.sh
 
 [testenv:functional]
-commands = {toxinidir}/tools/test/functest.sh
+allowlist_externals = bash
+commands = bash {toxinidir}/tools/test/functest.sh
 
 [testenv:docs]
 commands = sphinx-build -j auto -d {toxinidir}/doc/build/doctrees -b html {toxinidir}/doc/source doc/build/html
```

Comment:
Interesting that this env does not require an allowlist.

Reply:
I think there are some commands that are implicitly allowed, like stestr and this one, and you only find out when you run it using tox4 :)

Reply:
Haha, always great to find those obscure exceptions 😉